e2eTests/e2eTestPrompts.md

```diff
@@ -1,52 +1,82 @@
-| azmcp-cosmos-database-list | List all the database in my cosmosdb account <account_name> |
+| azmcp-extension-az | List my resource groups using the Azure CLI |
+| azmcp-extension-az | Show me the details of the storage account <account_name> using the Azure CLI |
+| azmcp-extension-az | List all virtual machines in my subscription using the Azure CLI |
 
-## Key Vault
+## Azure Cosmos DB
 
 | Tool Name | Test Prompt |
 |:----------|:----------|
-| azmcp-keyvault-key-create | Create a new key called <key_name> with RSA type in <key_vault_account_name> |
-| azmcp-keyvault-key-get | Get the details about the key <key_name> in <key_vault_account_name> |
-| azmcp-keyvault-key-list | List all the keys in <key_vault_account_name> |
+| azmcp-cosmos-account-list | List all cosmosdb accounts in my subscription |
+| azmcp-cosmos-database-list | List all the databases in the cosmosdb account <account_name> |
+| azmcp-cosmos-database-container-list | List all the containers in the database <database_name> for the cosmosdb account <account_name> |
+| azmcp-cosmos-database-container-item-query | Show me the items that contain the word <search_term> in the container <container_name> in the database <database_name> for the cosmosdb account <account_name> |
 
 ## Azure Data Explorer
 
 | Tool Name | Test Prompt |
 |:----------|:----------|
-| | |
+| azmcp-kusto-cluster-list | List all Data Explorer clusters in my subscription |
+| azmcp-kusto-cluster-get | Show me the details of the Data Explorer cluster <cluster_name> |
+| azmcp-kusto-database-list | List all databases in the Data Explorer cluster <cluster_name> |
+| azmcp-kusto-table-list | List all tables in the Data Explorer database <database_name> in cluster <cluster_name> |
+| azmcp-kusto-table-schema | Show me the schema for table <table_name> in the Data Explorer database <database_name> in cluster <cluster_name> |
+| azmcp-kusto-query | Show me all items that contain the word <search_term> in the Data Explorer table <table_name> in cluster <cluster_name> |
+| azmcp-kusto-sample | Get a sample of data from the Data Explorer table <table_name> in cluster <cluster_name> |
 
-## Monitor
+## Azure Key Vault
 
 | Tool Name | Test Prompt |
 |:----------|:----------|
+| azmcp-keyvault-key-list | List all the keys in the key vault <key_vault_account_name> |
+| azmcp-keyvault-key-get | Get the details about the key <key_name> in the key vault <key_vault_account_name> |
+| azmcp-keyvault-key-create | Create a new key called <key_name> with the RSA type in the key vault <key_vault_account_name> |
+| azmcp-keyvault-secret-get | Get the details about the secret <secret_name> in the key vault <key_vault_account_name> |
+
+## Azure Monitor
+
+| Tool Name | Test Prompt |
+|:----------|:----------|
+| azmcp-monitor-workspace-list | List all Log Analytics workspaces in my subscription |
```

Comment:
Nit: It would be nice to organize the table by tool name in alphabetical order.

Reply:
@fanyang-mono Not speaking about Azure Monitor but other prompts I added in this PR: In my mind it made sense to group things from general to specific, like "list storage accounts" to "list containers in storage account x" and then "list blobs in container x". Would that help with your manual testing or should we do strictly alphabetical ordering?

Reply:
Thanks for thinking about testing convenience. Organizing the commands by alphabetical order should be sufficient.